### DISCUSSION TITLE
Multiple Flags artist country

### DISCUSSION THREAD

**Discussion by giorgiobrit** [TT-ReBORN/Georgia-ReBORN#105] [Q&A] [answered]
Hello, how can i insert two flags ? i tried in the tag ARTISTCOUNTRY to insert for example : Wales, United Kigdom but it doesn't work.

Anyone help me?
thanks

**Comment by silverwings15** [comment, marked as answer]
select songs, right click, Properties, right click on ArtistCountry tag and Split Values 
![image](https://user-images.githubusercontent.com/10742312/224687664-061b9368-1d93-4a1e-9fcd-3a966b81ec81.png)

**Reply by giorgiobrit** [reply]
Thank you very much!
